client/components/sidebar/sidebarArchives.js

```diff
@@ -11,11 +11,13 @@ BlazeComponent.extendComponent({
   },
 
   archivedCards() {
-    return Cards.find({ archived: true });
+    const board = Boards.findOne(Session.get('currentBoard'));
```

Comment:
You don’t need to fetch the entire board document here since only its `_id` is used.